src/main/cliapp/src/containers/dataLoadsPage/DataLoadsComponent.js

```diff
@@ -59,14 +59,14 @@ export const DataLoadsComponent = () => {
 	};
 	
 	const loadTypeClasses = new Map([
-		["FULL_INGEST", ["GeneDiseaseAnnotation", "AlleleDiseaseAnnotation", "AGMDiseaseAnnotation", "Gene", "Allele", "AffectedGenomicModel"]],
-		["DISEASE_ANNOTATION", ["GeneDiseaseAnnotation", "AlleleDiseaseAnnotation", "AGMDiseaseAnnotation"]],
-		["GENE_DISEASE_ANNOTATION", ["GeneDiseaseAnnotation"]],
-		["ALLELE_DISEASE_ANNOTATION", ["AlleleDiseaseAnnotation"]],
-		["AGM_DISEASE_ANNOTATION", ["AGMDiseaseAnnotation"]],
-		["GENE", ["Gene"]],
-		["ALLELE", ["Allele"]],
-		["AGM", ["AffectedGenomicModel"]]
+		["FULL_INGEST", ["GeneDiseaseAnnotationDTO", "AlleleDiseaseAnnotationDTO", "AGMDiseaseAnnotationDTO", "GeneDTO", "AlleleDTO", "AffectedGenomicModelDTO"]],
+		["DISEASE_ANNOTATION", ["GeneDiseaseAnnotationDTO", "AlleleDiseaseAnnotationDTO", "AGMDiseaseAnnotationDTO"]],
+		["GENE_DISEASE_ANNOTATION", ["GeneDiseaseAnnotationDTO"]],
+		["ALLELE_DISEASE_ANNOTATION", ["AlleleDiseaseAnnotationDTO"]],
+		["AGM_DISEASE_ANNOTATION", ["AGMDiseaseAnnotationDTO"]],
+		["GENE", ["GeneDTO"]],
+		["ALLELE", ["AlleleDTO"]],
+		["AGM", ["AffectedGenomicModelDTO"]]
```

Comment:
Just wondering if the data that is in the Load tables need to be changed? I didn't see any of those in the migration.

Reply:
Just looked it up its the type thats getting stored not the class, eg (GENE, ALLELE, etc).

Reply:
Yeah, so this should be good as is.  Uses type to lookup appropriate DTO class schema versions.